src/astro/sql/operators/append.py

```diff
@@ -54,3 +54,28 @@ def execute(self, context: dict) -> Table:  # skipcq: PYL-W0613
             source_to_target_columns_map=self.columns,
         )
         return self.target_table
+
+
+def append(
+    *,
+    source_table: Table,
+    target_table: Table,
+    columns: APPEND_COLUMN_TYPE = None,
+    **kwargs: Any,
+):
```

Comment:
Let's please add the return type for these functions.

Reply:
Done in https://github.com/astronomer/astro-sdk/pull/637/commits/98fd8a561cb136b865525b33b5b1207dffd1d122